tests/unit/sharded_copying_manager_tests/common.py

```diff
@@ -427,18 +413,13 @@ class TestableCopyingManagerThreadedWorker(
     Doing this right is a bit complicated because the CopyingManagerWorker runs in its own thread.
 
     Many members of this class are common with the TestableCopyingManager class
-    and mare oved to the base class - TestableCopyingManagerFlowController
+    and moved to the base class - TestableCopyingManagerFlowController
     """
 
     __test__ = False
 
-    # The different points at which the CopyingManager can be stopped.  See below.
-    SLEEPING = "SLEEPING"
-    SENDING = "SENDING"
-    RESPONDING = "RESPONDING"
-
     # To prevent tests from hanging indefinitely, wait a maximum amount of time before giving up on some test condition.
-    WAIT_TIMEOUT = 5.0
+    WAIT_TIMEOUT = 500.0
```

Comment:
Ok, this seems long.  Can we set it to 60?

Reply:
5.0 should be fine. I guess this one was outdated because it's 5 now.